extensions/wikia/AdEngine/tests/AdEngine2ContextServiceTest.php

```diff
@@ -45,84 +45,84 @@ private function mockMonetizationModule() {
+				'expectedOpts' => [ 'enableAdsInMaps' => true ]
 			],
 			[
 				'titleMockType' => 'article',
-				'flags' => ['wgAdDriverEnableInvisibleHighImpactSlot'],
-				'expectedOpts' => [],
+				'flags' => [ 'wgAdDriverEnableInvisibleHighImpactSlot' ],
+				'expectedOpts' => [ ],
 				'expectedTargeting' => [ 'newWikiCategories' => [ 'test' ] ],
-				'expectedProviders' => [],
+				'expectedProviders' => [ ],
 				'expectedForceProviders' => null,
-				'expectedSlots' => ['invisibleHighImpact' => true]
+				'expectedSlots' => [ 'invisibleHighImpact' => true ]
 			],
 			[
 				'titleMockType' => 'article',
-				'flags' => ['wgAdDriverForceTurtleAd'],
-				'expectedOpts' => [],
+				'flags' => [ 'wgAdDriverForceTurtleAd' ],
+				'expectedOpts' => [ ],
 				'expectedTargeting' => [ 'newWikiCategories' => [ 'test' ] ],
-				'expectedProviders' => [],
+				'expectedProviders' => [ ],
 				'expectedForcedProvider' => 'turtle'
 			],
 			[
 				'titleMockType' => 'article',
-				'flags' => ['wgAdDriverTrackState'],
-				'expectedOpts' => ['trackSlotState' => true],
+				'flags' => [ 'wgAdDriverTrackState' ],
+				'expectedOpts' => [ 'trackSlotState' => true ],
 				'expectedTargeting' => [ 'newWikiCategories' => [ 'test' ] ]
 			],
 			[
 				'titleMockType' => 'article',
-				'flags' => ['wgAdDriverWikiIsTop1000'],
-				'expectedOpts' => [],
+				'flags' => [ 'wgAdDriverWikiIsTop1000' ],
+				'expectedOpts' => [ ],
 				'expectedTargeting' => [ 'newWikiCategories' => [ 'test' ], 'wikiIsTop1000' => true ]
 			],
 			[
 				'titleMockType' => 'article',
-				'flags' => ['wgEnableAdsInContent'],
-				'expectedOpts' => ['adsInContent' => true]
+				'flags' => [ 'wgEnableAdsInContent' ],
+				'expectedOpts' => [ 'adsInContent' => true ]
 			],
 			[
 				'titleMockType' => 'article',
-				'flags' => ['wgEnableOutboundScreenExt'],
-				'expectedOpts' => [],
+				'flags' => [ 'wgEnableOutboundScreenExt' ],
+				'expectedOpts' => [ ],
 				'expectedTargeting' => [ 'newWikiCategories' => [ 'test' ] ],
-				'expectedProviders' => [],
+				'expectedProviders' => [ ],
 				'expectedForceProviders' => null,
-				'expectedSlots' => ['exitstitial' => true]
+				'expectedSlots' => [ 'exitstitial' => true ]
 			],
 			[
 				'titleMockType' => 'article',
-				'flags' => ['wgOutboundScreenRedirectDelay'],
-				'expectedOpts' => [],
+				'flags' => [ 'wgOutboundScreenRedirectDelay' ],
+				'expectedOpts' => [ ],
 				'expectedTargeting' => [ 'newWikiCategories' => [ 'test' ] ],
-				'expectedProviders' => [],
+				'expectedProviders' => [ ],
 				'expectedForceProviders' => null,
-				'expectedSlots' => ['exitstitialRedirectDelay' => true]
+				'expectedSlots' => [ 'exitstitialRedirectDelay' => true ]
 			],
 			[
 				'titleMockType' => 'article',
-				'flags' => ['wgEnableWikiaHomePageExt'],
-				'expectedOpts' => ['pageType' => 'corporate'],
+				'flags' => [ 'wgEnableWikiaHomePageExt' ],
+				'expectedOpts' => [ 'pageType' => 'corporate' ],
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "pageType" 8 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)